README.md

```diff
@@ -53,6 +53,10 @@ Have a look at the [examples directory](examples) for some use cases
 - `data` (string, required): Valid JSON data that this provider will manage with the API server. This should represent the whole API object that you want to create. The provider's information.
 - `force_new` (array of strings, optional): Any changes to these values will result in recreating the resource instead of updating.
 - `debug` (boolean, optional): Whether to emit verbose debug output while working with the API object on the server. This can be gathered by setting `TF_LOG=1` environment variable.
+- `read_search` (bool, optional): Enables usage of `search_key` and `search_value` or `results_key` on `read_path` only.
```

Comment:
Pursuing a duplication of what the datasource does, this part of the readme can be modified to just say `read_search` is a key that holds the keys that the datasource has (so as to not duplicate the docs)